app/mailers/candidate_mailer.rb

```diff
@@ -304,7 +304,7 @@ def changed_offer(application_choice)
 
   def change_course(application_choice)
     @application_choice = application_choice
-    @course_option = @application_choice.course_option
+    @course_option = @application_choice.original_course_option
```

Comment:
Having a think about this, I don't think we can use `original_course_option` here. This is because using the original course option is only true the first time, if we change the course multiple times this no longer holds the correct information. I think we will need to pass into the `change_course` mailer method another variable: `old_course` which we have access to in the service before persisting the new course

Reply:
🆗